go.mod

```diff
@@ -88,3 +88,5 @@ require (
 	gopkg.in/ini.v1 v1.67.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/couchbase/goprotostellar => ../goprotostellar
```

Comment:
This from local testing?

Reply:
Yea.  Needs code merged to protostellar first.